src/constants/constants.js

```diff
@@ -1,13 +1,6 @@
-export const options = ['Russia', 'Belarus', 'Ukraine'];
+export const sorts = ['popularity', 'relevancy', 'publishedAt'];
 
-export const genders = ['male', 'female'];
+export const minPageSize = '2';
```

Comment:
For default variables, it`s better to use something like MIN_PAGE_SIZE (snake case), to separate from other variables visually